### PR TITLE
add interface to Conway polys in flint

### PR DESCRIPTION
**Pull request by fieker**
not quite sure if this should be redone to look more like a database...

### CODE REVIEW

File: examples/Conway.jl [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  if unsafe_load(c, i) == 0 ||
+     unsafe_load(c, i+1) != n
+     return false
+  end
+  return true
+end
+
+@doc Markdown.doc"""
+    conway_polynomials(p::Int)
+Returns an array with all Conway polynomials for the given prime `p`.
```
Comment (ThomasBreuer) [inline examples/Conway.jl]: Return an array with all known Conway polynomials for the given prime `p`.

File: examples/Conway.jl [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  if unsafe_load(c, i) == 0 ||
+     unsafe_load(c, i+1) != n
+     error("no conway polynomial known")
+  end
+  return polynomial(GF(p), [unsafe_load(c, j) for j=i+2:i+2+unsafe_load(c, i+1)+1])
+end
+
+@doc Markdown.doc"""
+    hasconway_polynomial(p::Int, n::Int)
+Tests if the Conway polynomial of degree `n` is known for this prime.
```
Comment (ThomasBreuer) [inline examples/Conway.jl]: Test if the Conway polynomial of degree `n` is known for the prime `p`.

File: examples/Conway.jl [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  i = 1
+  while unsafe_load(c, i) != 0 && unsafe_load(c, i) < p
+    i += unsafe_load(c, i+1)+3 #prime, degree, coeffs (deg+1 many)
+  end
+  while unsafe_load(c, i) != 0 && unsafe_load(c, i+1) < n
+    i += unsafe_load(c, i+1)+3
+  end
+  if unsafe_load(c, i) == 0 ||
+     unsafe_load(c, i+1) != n
+     error("no conway polynomial known")
```
Comment (ThomasBreuer) [inline examples/Conway.jl]: `error("Conway polynomial not known")`

Comment (thofma) [inline examples/Conway.jl]:
```suggestion
     error("Conway polynomial not known")
```

File: examples/Conway.jl (hunk 2 above)
Comment (thofma) [inline examples/Conway.jl]:
```suggestion
Test if the Conway polynomial of degree `n` is known for the prime `p`.
```

File: examples/Conway.jl (hunk 1 above)
Comment (thofma) [inline examples/Conway.jl]:
```suggestion
Return an array with all Conway polynomials for the given prime `p`.
```

**Comment by thofma** [comment]
@ThomasBreuer Maybe next time you could use the "suggestion" version when doing the comments, then we can commit the suggested changes directly.

**Review by tthsqe12 — APPROVED** [review]
Looks good. I am surprised by how many polynomials are actually stored there.

**Pull request merged**